src/internal/views/nunjucks/view-licences/tabs/charge.njk

```diff
@@ -14,19 +14,23 @@
   {# {% if not agreement.dateRange.endDate %}
     <a href='/licences/{licenceId}/agreements/{{ agreement.id }}/end'>End</a> 
   {% endif %} #}
-  {% if isChargingUser %} 
-    <a href='/licences/{{ licenceId }}/agreements/{{ agreement.id }}/delete'>Delete</a> 
+  {% if isChargingUser %}
+    <a href='/licences/{{ licenceId }}/agreements/{{ agreement.id }}/delete'>Delete</a>
   {% endif %}
 {% endmacro %}
 
 {% macro chargeVersionRowMobile(chargeVersion) %}
   <div class="phone--show">
     <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
 
-    <p>Start date: {{ chargeVersion.dateRange.startDate | date }}</p>
-    <p>End date: {{ chargeVersion.dateRange.endDate | date }}</p>
-    <p>Reason: </p>
-    <p>Status: {{ badge(chargeVersion | chargeVersionBadge) }}</p>
+    <p>Start date:
+      {{ chargeVersion.dateRange.startDate | date }}</p>
+    <p>End date:
+      {{ chargeVersion.dateRange.endDate | date }}</p>
+    <p>Reason:
```

Comment:
Reason here doesn't have a value. Ignore if this is intentional.

Reply:
I've added this to ticket 2845 which is the charge information wrap-up